haproxy/src/configure.py

```diff
@@ -46,8 +50,8 @@
 
 frontend_conf = Template("""
   frontend $name
-    bind *:$port $accept_proxy
-    mode $mode
+    bind *:$port $accept_proxy$ssl_bind
```

Comment:
I don't see how this approach helps me to add optional lines in the middle of the template. Your example appends to the end of processed template but not in the middle.

The root problem is that string.Template is not very powerful so I did this workaround. It would be better to have a full-fledged template engine like Jinja2.